strawberry/http/base.py

```diff
@@ -67,6 +67,15 @@ def parse_query_params(
             if variables:
                 params["variables"] = self.parse_json(variables)
 
+        if "extensions" in params:
+            extensions = params["extensions"]
+
+            if isinstance(extensions, list):
+                extensions = extensions[0]
```

Comment:
Why only getting the first on the list?

From the rest of the code `extensions` should be a dict. When can it be a list? If it is, is there a possibility of it having more than one value inside it? If yes, what does that mean? Should we warn/error out about that?

Reply:
Since it has the exact same type definition as the `variables` parameter, I just copy pasted how that is being parsed. I think it might be a list in GET requests?

Reply:
Hrmm I see. @strawberry-graphql/core do you know why we do that?

Reply:
I just dug around a little bit. Before #2681 was merged there used to be the following comment in the Sanic integration:

https://github.com/strawberry-graphql/strawberry/blob/860e5f236b936440d4d28b8cea3f7bc9316a1612/strawberry/sanic/views.py#L101-L108

This could explain why we parse variables in that way.

Reply:
Update: I'm working on a PR for this. Looks like it might be Sanics fault after all.

Reply:
#3558